src/main/resources/GoogleApps/Translations.xml

```diff
@@ -39,30 +39,49 @@
   <minorEdit>false</minorEdit>
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
-  <content><![CDATA[
-
+  <content>
 admin.googleapps=Google Apps
 googleapps.config.heading=Google Apps Integration Configuration
-googleapps.config.explanation=XWiki can be integrated with Google Apps, which then allows you to edit Office Attachments using the Google Apps editors as well as resave them in the wiki. It also allows to upload a Google Apps file to any Wiki Page. To activate Google Apps you need to [[register for OAuth access on Google.com>>https://accounts.google.com/ManageDomains]] and to insert your API-key and secret in the following settings. For more information, please see [[the installation instructions>>https://store.xwiki.com/xwiki/bin/view/Extension/GoogleAppsIntegration#installation]].
-GoogleApps.GoogleAppsConfigClass_active=Activate
+googleapps.config.explanation=XWiki can be integrated with Google Apps, which then allows you to edit Office Attachments using the Google Apps editors as well as resave them in the wiki. It also allows to upload a Google Apps file to any Wiki Page.
+GoogleApps.GoogleAppsConfigClass_activate=Activate
+
+GoogleApps.GoogleAppsConfigClass_communicate=Communication with Google services
+GoogleApps.GoogleAppsConfigClass_communicate.hint=To activate Google Apps you need to {0}register for OAuth access on the Google Cloud console{1} and to insert your Client-Id and secret in the following settings.
+GoogleApps.GoogleAppsConfigClass_communicate.hint2=For more information, please see {0}the installation instructions{1}.
+
 GoogleApps.GoogleAppsConfigClass_appname=Application Name
-GoogleApps.GoogleAppsConfigClass_appname.hint=For your own organisation.
+GoogleApps.GoogleAppsConfigClass_appname.hint=For your own organisation..
```

Comment:
Is the second `.` needed for this translation?

Reply:
fixed. Thanks.